omf-bifrost/overture-valhalla-writer/src/writer.rs

```diff
@@ -307,6 +326,23 @@ fn export_roads(exported_roads: &[ExportedRoad], output_dir: &Path) -> std::io::
     Ok(())
 }
 
+fn check_permissions(road_class: &str) -> Permissions {
+    let pedestrian_allowed = !matches!(
+        road_class,
+        "motorway" | "trunk" | "cycleway" | "standard_gauge"
+    );
+
+    let auto_allowed = !matches!(
+        road_class,
+        "null" | "track" | "steps" | "path" | "living_street" | "pedestrian" | "footway" | "cycleway" | "standard_gauge"
```

Comment:
nit: Track may need to be allowed based on the [wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrack)